### ISSUE TITLE
Feature request: Unlimited number of recent items

### ISSUE THREAD

**Issue by SwallowYourDreams** [TomBursch/kitchenowl#471]
### What's the feature 🧐

Recent items (i.e. items that are not on the shopping list, but have been recently) should no longer be limited to a maximum of 30. It should be possible to dial up the number so high that users could see everything they've ever had on the shopping list. It'd allow the user to look at their "default purchases", i.e. things that they regularly buy, for inspiration, and tick off anything they might have forgotten.

**Comment by RHuehne** [comment]
yes, that would be nice. Additionally I would love to have added the feature, to have the recent items list in a list view. I am doing most of my KitchenOwl interactions on classical devices, and only use touch in the shop. Therefore the tales view "eats" a lot of content on the screen.

**Comment by DanteMS** [comment]
Would be great to have this feature. It's a bit annoying to search every time for stuff I regularly buy or to forget to buy something because it disappeared from the list of recent items.
